src/Categories/NaturalTransformation/Extranatural.agda

```diff
@@ -0,0 +1,52 @@
+{-# OPTIONS --without-K --safe #-}
+module Categories.NaturalTransformation.Extranatural where
+
+-- Although there is a notion of Extranatural in Categories.NaturalTransformation.Dinatural,
```

Comment:
it looks somewhat close to dinaturality. I think dinatural is a special base of this definition. do you have a source of this definition?

Reply:
I am following the coend calculus book, page-by-page.  Yes, dinaturality is a special case, but the proof is a bit complicated (I skipped that).